shapely/tests/legacy/test_get_segments.py

```diff
@@ -0,0 +1,40 @@
+import numpy
```

Comment:
This is a convention that you'll find throughout the repo:

```suggestion
import numpy as np
```